caiman/cluster.py

```diff
@@ -34,27 +34,13 @@
 import subprocess
 import sys
 import time
+from typing import Any, Dict, List, Optional, Tuple, Union
 
 from .mmapping import load_memmap
 
 logger = logging.getLogger(__name__)
 
-def get_patches_from_image(img, shapes, overlaps):
```

Comment:
@pgunn Why is this method being removed?

Reply:
Was unused. If that's not a reason to remove it I can revert this part. (same as below)